components/CommunityModerators/CommunityModerators.module.css

```diff
@@ -0,0 +1,23 @@
+.container {
+    @apply bg-[#050B1E] flex flex-col gap-[65px] justify-center items-center pt-20 pb-24 pl-4 pr-4;
```

Comment:
`#050B1E` is actually defined as `primary-blue-400`

Reply:
i fixed it